### ISSUE TITLE
Fix table conversion

### ISSUE THREAD

**Issue by frauzufall** [imagej/imagej.github.io#24]
Many tables become unusable during conversion with a good part of a table being completely gone or rows not matching up any more. I have not looked into it and I have not been able to spot a pattern, but here is an example:

Old: https://imagej.net/Distribution
New: http://imagej.github.io/pages/Distribution

**Comment by ctrueden** [comment]
The issue is caused by tables with a `colspan`, especially one on the first row of the table. It appears that pandoc counts the number of columns in the first row, and then assumes all rows will have the same number of columns... and ignores columns beyond this count.

A potential fix would be to preprocess the MediaWiki sources like this:
```diff
-| colspan="4" style="background: #afa" | '''Create your own update site'''
+| '''Create your own update site''' <a href='CSS-TOREMOVE: colspan="4" style="background: #afa'></a>
```
which will improve pandoc's output as follows:
```diff
-<td><p><strong>Create your own update site</strong></p></td>
+<td><p><strong>Create your own update site</strong> <a href='CSS-TOREMOVE: colspan="4" style="background: #afa'></a></p></td>
```
And then we can more easily post-process the `CSS-TOREMOVE` part back into styles and colspans.

**Comment by frauzufall** [comment]
Made progress, most of the data should be there now, the MediaWiki table CSS styling too in case it's converted to a HTML table. otherwise it currently gets deleted. But there are still issues:
- bigger issue: the includes in the tables don't work (e.g. https://imagej.github.io/BigDataViewer)
- smaller issue: Any CSS HTML styling on MediaWiki pages is removed by pandoc - not sure why, but this makes some tables with global row styling and individual content cell styling not look as expected, e.g. https://imagej.github.io/Development
- smaller issue: in case the tables are not converted into HTML by pandoc, the markdown tables don't align any more after removing the styling from it. This makes no difference for rendering, but the source code looks less nice

**Comment by frauzufall** [comment]
Includes in tables are fixed with aebdc17e506763b794bb57de03c51487e11f00c7 by forcing such a table to be converted as HTML table and not markdown table.

**Comment by frauzufall** [comment]
I tried to keep the CSS styling for other HTML tags ([branch](https://github.com/imagej/imagej.github.io/tree/keep-styles)) but it doesn't work great. In case no one screams that this NEEDS to be fixed, I will not continue to work on this, just to get to an end at some point. 

Additionally, these issues are still open regarding tables, but I will move on:
- in case there is styling in the table but the table is converted to a pure markdown table, the styling is gone
- the alignment of the table code in markdown is sometimes not pretty (because we add and later (after pandoc is done) remove hidden styling code)

Hopefully, the rest is ok.
